alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -487,6 +489,89 @@ fn box_drawing(character: char, metrics: &Metrics, offset: &Delta<i8>) -> Raster
             // Fourth quadrant.
             canvas.draw_rect(x_center, y_center, w_fourth, h_fourth, COLOR_FILL);
         },
+        // Sextants: '🬀', '🬁', '🬂', '🬃', '🬄', '🬅', '🬆', '🬇', '🬈', '🬉', '🬊', '🬋', '🬌', '🬍', '🬎',
+        // '🬏', '🬐', '🬑', '🬒', '🬓', '🬔', '🬕', '🬖', '🬗', '🬘', '🬙', '🬚', '🬛', '🬜', '🬝', '🬞', '🬟',
+        // '🬠', '🬡', '🬢', '🬣', '🬤', '🬥', '🬦', '🬧', '🬨', '🬩', '🬪', '🬫', '🬬', '🬭', '🬮', '🬯', '🬰',
+        // '🬱', '🬲', '🬳', '🬴', '🬵', '🬶', '🬷', '🬸', '🬹', '🬺', '🬻'.
+        '\u{1fb00}'..='\u{1fb3b}' => {
+            let x_center = canvas.x_center().round().max(1.);
+            let y_third = (height as f32 / 3.).round().max(1.);
+            let y_last_third = height as f32 - 2. * y_third;
+
+            let (w_1, h_1) = match character {
```

Comment:
Could you name the variables based on the location in the shape, like `top-left`, `top-right`, `mid-left`, etc?